hack/make/.integration-test-helpers

```diff
@@ -64,8 +64,13 @@ run_test_integration() {
 }
 
 run_test_integration_suites() {
-	local flags="-test.v -test.timeout=${TIMEOUT} $TESTFLAGS"
 	local dirs="$1"
+	local flags="-test.v -test.timeout=${TIMEOUT} $TESTFLAGS "
```

Comment:
> We probably need to check if TEST_FILTER is not empty in that case (I didn't add that to my example)

🙈 guess you overlooked that part https://github.com/moby/moby/pull/46169#discussion_r1287108455 😂 

```
[2023-08-08T14:55:10.735Z] INFO: Testing against a local daemon
[2023-08-08T14:55:10.735Z] flag needs an argument: -test.run
```

Reply:
🤦🏻  I remembered about it, but I was too amazed with how much not having these additional env vars is better, that I actually forgot to do it 🤣 
Fixed!